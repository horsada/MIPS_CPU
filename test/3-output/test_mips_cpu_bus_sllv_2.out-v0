FINAL OUT: a2000000
