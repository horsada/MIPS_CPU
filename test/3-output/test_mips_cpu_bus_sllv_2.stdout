WARNING: mips_cpu_ram.v:24: $readmemh: Standard inconsistency, following 1364-2005.
VCD info: dumpfile mips_cpu_bus_tb.vcd opened for output.
FINAL OUT: a2000000
